integration_tests/e2e/personal/edit/alias/addNewAlias.cy.ts

```diff
@@ -22,6 +22,7 @@ context('Add new alias', () => {
         prisonerNumber,
         response: PseudonymResponseMock,
       })
+      cy.task('stubAllPersonalCareNeeds')
```

Comment:
It would be better to add this stub to `setupPersonalPageStubs` (see commands.ts) which is already called rather than having to add this everywhere `setupPersonalPageStubs` is called.